specifications/transactions/offchain-api/README.md

```diff
@@ -0,0 +1,256 @@
+
+## Overview
+The two parties that participate in the off-chain protocol communicate through HTTP requests and responses within a TLS channel. Each request and response is signed via [JSON Web Signature (JWS)](https://tools.ietf.org/html/rfc7515) to ensure authenticity and integrity. Consider an example in which VASP A sends a command to VASP B:
+
+* VASP A knows VASP B’s on-chain account and obtains the url from VASP B's `DualAttestation::Credential::base_url` resource
+* VASP A connects to this endpoint and establishes a tls-encrypted HTTP channel
+* VASP A sends a request across the channel
+  * VASP A, first, constructs message containing a `CommandRequestObject`
+  * VASP A, then, signs the message it with the Ed25519 private key that maps to the public key stored at VASP A’s `DualAttestation::Credential::compliance_public_key`
+  * VASP A, finally transmits the signed message across the channel
+* VASP B receives the `CommandRequestObject`
+  * VASP B, first, validates the request message
+  * VASP B, then, evaluates and responds to the message constructing `CommandResponseObject`
+  * VASP B signs the response with the Ed25519 private key that maps to the public key stored at VASP B’s `DualAttestation::Credential::compliance_public_key`
+* VASP A receives the `CommandResponseObject`, validates the response, and concludes the exchange
+
+## Specification
+### On-Chain Account Setup
+
+A VASP’s on-chain account must be created and set up in accordance with the standard VASP account creation process.
+
+Of particular note for off-chain APIs is the compliance key and base url, [DualAttestation::Credential](https://github.com/diem/diem/blob/release-1.1/language/stdlib/modules/DualAttestation.move)’s `compliance_public_key` and `base_url`. During account setup, a VASP should set their compliance key and base url which will be stored under the Parent VASP account, see [DIP-2](https://github.com/diem/dip/blob/master/dips/dip-2.md) for more information on on-chain roles.
+
+The compliance key value is the Ed25519 key with which a VASP signs travel rule statements which are verified on-chain and all off-chain requests and responses.
+
+The `base_url` frameowrk does not support multiplexing and should not be shared across different providers and accounts.
```

Comment:
```suggestion
The `base_url` framework does not support multiplexing and should not be shared across different providers and accounts.
```

Reply:
What is a "base_url" framework? This could be better worded if the intent is that the the base_url should be unique for different accounts and not shared.

Reply:
ahh, yes